compiler/src/typed/env.re

```diff
@@ -714,27 +714,14 @@ let add_import = s => {
   imported_units := StringSet.add(s, imported_units^);
 };
 
-let imported_opaque_units = ref(StringSet.empty);
```

Comment:
Could you explain why you removed these opaque unit imports?

Reply:
It's just old cruft from back in the day.